api/api-doc/compaction_manager.json

```diff
@@ -244,6 +262,23 @@
             }
          }
       },
+      "pending_compaction": {
+        "id":"pending_compaction",
+        "properties":{
+            "cf":{
+               "type":"string",
```

Comment:
you forgot many space after ":" here

Reply:
sorry, I did the change and didn't push it.

I've forced push the changes now